src/Pangraph/ProtoGraph.hs

```diff
@@ -0,0 +1,87 @@
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
It would be nice to add some top-level module documentation to describe the purpose of various modules.